src/preferences/general.hpp

```diff
@@ -274,6 +274,9 @@ namespace preferences {
 	sort_order::type addon_manager_saved_order_direction();
 	void set_addon_manager_saved_order_direction(sort_order::type value);
 
+	std::string selected_achievement_group();
+	void set_selected_achievement_group(const std::string& id);
```

Comment:
The naming of `id` feels a bit weird, because IIUC other functions call this `content_for` and call something else `id`.

Reply:
Yeah, true, updated.